partition/roles/bmc-proxy/tasks/main.yaml

```
@@ -0,0 +1,104 @@
+---
+- name: Check mandatory variables for this role are set
+  assert:
+    fail_msg: "not all mandatory variables given, check role documentation"
+    quiet: yes
+    that:
+      - bmc_proxy_image_name is defined
+      - bmc_proxy_image_tag is defined
+      - bmc_reverse_proxy_image_name is defined
+      - bmc_reverse_proxy_image_tag is defined
```

```suggestion
      - bmc_reverse_proxy_image_tag is defined
      - bmc_proxy_certs_ca_cert is not none
      - bmc_proxy_certs_server_cert is not none
      - bmc_proxy_certs_server_key is not none
```

This should be done for mandatory variables, can we also do this in the metal role?